packages/styles/scss/components/accordion/_accordion.scss

```diff
@@ -154,11 +153,18 @@ $content-padding: 0 0 0 $spacing-05 !default;
     text-align: start;
   }
 
+  .#{$prefix}--accordion__wrapper {
+    display: grid;
```

Comment:
I'm not confident that this won't break existing things people place in this component that inherently rely on the previous `display: block`. We could try, but I think it increases the risk of needing to roll this back if we publish it and get feedback from teams that this has unintended consequences.

Reply:
I have resolved this issue by using another approach instead of CSS grid